tests/Common/TokensTest.php

```diff
@@ -505,4 +505,70 @@ public function testExpiredToken()
         }
         $this->fail('token should be invalid');
     }
+
+    public function testTokenUpdateKeepsCanManageBucketsFlag()
+    {
+        $bucketIds = array_map(
+            function ($bucket) {
+                return $bucket['id'];
+            },
+            $this->_client->listBuckets()
+        );
+
+        $this->assertGreaterThan(0, count($bucketIds));
+
+        $tokenId = $this->_client->createToken(
+            'manage',
```

Comment:
Tady mě trochu zmátlo, že první parametr má být array a je to string. Ale koukám, že tam pak je nějaký hack.